tiatoolbox/wsicore/wsireader.py

```diff
@@ -18,10 +18,13 @@
 import tifffile
 import zarr
 from defusedxml import ElementTree
+from PIL import Image
 
 from tiatoolbox import utils
+from tiatoolbox.annotation.storage import AnnotationStore, SQLiteStore
```

Comment:
Did you get a chance to check for circular imports?

Reply:
As far as I can see there are no circular imports